app/src/App.js

```diff
@@ -59,12 +59,15 @@ class App extends Component {
         </div>
         <main>
           <form
-            autocomplete="off"
+              autoComplete="off"
+              autoCorrect="off"
+              autoCapitalize="off"
+              spellCheck="false"
             className="form"
             onSubmit={this.handleSubmit}
           >
             <FormGroup>
-              <ControlLabel>Main key</ControlLabel>
+              <FormLabel>Main key</FormLabel>
```

Comment:
This changes the way the label text is rendered, because now the element uses `.form-label` CSS class, instead of `.control-label`. `App.css` file defines rules for `.control-label` class, which currently style this element and make it big, bold and readable. Could you please also update the `App.css` file by renaming the `.form-label` class it defines to `.control-label`? 🙏  This would make the change more consistent by keeping the old styling intact and making the form label more readable

Reply:
ControlLabel: the problem is that the react bootstrap team have deleted ControlLabel, it does not exist anymore,. Sure we can use any other label which is bigger, send an updated css now.

Reply:
pr of CSS done, test it and push it

Reply:
donations are welcome: (btc) 38hGkaU867SZDpAXBoZR1yfnXoipgo41om 
🤪